kbncontent.go

```diff
@@ -28,7 +28,7 @@ func (v VisualizationDescriptor) findDocumentPathsAsString(paths []string) strin
 	m := objx.Map(v.Doc)
 
 	for _, path := range paths {
-		if m.Get(path).IsStr() {
+		if m.Get(path).IsStr() && m.Get(path).Str() != "" {
```

Comment:
Without this change this seems to return the first path it founds, even if it is empty. There are cases where the first path is empty, but there are others with values.